.tool-versions

```diff
@@ -0,0 +1 @@
+ruby 3.1.1
```

Comment:
Any particular reason why not 3.1.2?

Reply:
Maybe... because I wrote that commit at 11pm? 😆